docs-ref-autogen/TOC.md

```diff
@@ -12,6 +12,7 @@
 ## [Output formats](../docs-ref-conceptual/format-output-azure-cli.md)
 ## [Query](../docs-ref-conceptual/query-azure-cli.md)
 ## [Subscriptions](../docs-ref-conceptual/manage-azure-subscriptions-azure-cli.md)
+## [Clouds](../docs-ref-conceptual/manage-cloud-sets-azure-cli.md)

```

Comment:
No need to update the autogen directory ToC manually in the future, but it doesn't harm anything by doing so. This file will get overwritten with an autogenerated splice merging the reference ToC with the ToC in docs-ref-conceptual.